lib/saml_idp/incoming_metadata.rb

```diff
@@ -18,26 +18,26 @@ def document
 
     def sign_assertions
       doc = xpath(
-        "//md:SPSSODescriptor",
+        '//md:SPSSODescriptor',
         ds: signature_namespace,
         md: metadata_namespace
       ).first
-      doc ? !!doc["WantAssertionsSigned"] : false
+      doc ? !!doc['WantAssertionsSigned'] : false
     end
     hashable :sign_assertions
 
     def display_name
-      role_descriptor_document.present? ? role_descriptor_document["ServiceDisplayName"] : ""
+      role_descriptor_document.present? ? role_descriptor_document['ServiceDisplayName'] : ''
     end
     hashable :display_name
 
     def contact_person
       {
-        given_name: given_name,
-        surname: surname,
-        company: company,
-        telephone_number: telephone_number,
-        email_address: email_address
+        given_name:,
```

Comment:
this is ruby 3.1 syntax i think? We should update the `.gemspec` to specify

```
spec.required_ruby_version = '>= 3.1.0'
```

Reply:
added in https://github.com/18F/saml_idp/pull/98/commits/2b8c1afa357e64a05ddf8932dfae209da2a40a6a